### ISSUE TITLE
Notify about updated articles

### ISSUE THREAD

**Issue by bcm314** [yanus171/Flym#131]
(auto)refresh feed also refreshes existing articles.
But when there is such an updated article, there is no notification.
How about an option to also notify about this?
So for example, you will be notified for each goal in a soccer game.

![Screenshot_2019-04-24-21-30-27](https://user-images.githubusercontent.com/30076623/56689935-83b49900-66dc-11e9-83a7-3d50fc81a113.png)
![Screenshot_2019-04-24-21-33-36](https://user-images.githubusercontent.com/30076623/56689951-8dd69780-66dc-11e9-97ba-f4cea02345d0.png)
![Screenshot_2019-04-24-21-51-11](https://user-images.githubusercontent.com/30076623/56689957-90d18800-66dc-11e9-9fba-aaf65c90e7a7.png)

**Comment by yanus171** [comment]
Yes, really? I thought that existing articles are not updated... I need to see the code... But your suggestion is sensable

**Comment by bcm314** [comment]
By the way: remember the (from me) closed issu https://github.com/yanus171/Flym/issues/136 ?

This is definitive a situation, where I like to have 'show read' on this feed (while not on 'normal news feeds')
But if also updated articles will be notified and switch back to unread ... ;-)

And for sure: existing articles are also update - I'm currently following a live-ticker.

**Comment by yanus171** [comment]
Yes, updated articles should be unread again

**Comment by ThomasLeigh** [comment]
@bcm314 , @yanus171 

How exactly this feature is supposed to work?

Does it refer to all the articles present within the app - or a user should tell the app which articles should be traced/checked occasionally for their updates?

**Comment by yanus171** [comment]
If we speak about One page mode, the article becomes unread again if it 
changes its date.

**Comment by ThomasLeigh** [comment]
1. Will this feature be implemented regarding One Page Mode?
2. Will this feature be implemented also regarding other articles (besides One Page Mode)?

**Comment by ChaosNicro** [comment]
Was looking for this too, for not having to manually check a website each day. Maybe a feed-type that the user selects on feed creation. It would cache a copy of the page and compare with the live version on refresh, creating a new entry from the live page when they differ. Would need a class-element filter to ignore ads, suggestion feeds and other dynamic things, in my case, I want to watch a list-container for changes.
